docker/nginx/web.conf

```diff
@@ -0,0 +1,22 @@
+server {
+    listen 80;
+    server_name docs.dev.readthedocs.io;
+
+    location /favicon.ico {
+        proxy_pass http://storage:10000/devstoreaccount1/static/images/favicon.ico;
+        break;
+    }
+
+    # FIXME: this is a workaround because CORS on the storage.
+    # The command to enable it is failing without reason:
+    # pip install azure-cli
+    # AZURE_STORAGE_CONNECTION_STRING="UseDevelopmentStorage=true" az storage cors add --origins 'docs.dev.readthedocs.io' --services 'b' --methods GET OPTIONS HEAD --allowed-headers '*'
```

Comment:
CORS it's not supported in Azurite yet: https://github.com/Azure/Azurite/issues/55#issuecomment-503380561